package.json

```diff
@@ -10,7 +10,6 @@
   "license": "MIT",
   "dependencies": {
     "@types/lodash": "^4.14.153",
```

Comment:
`lodash` is used in `src/typecasting.ts`. Nothing from that file is imported into `src/main.ts`, so `lodash` wouldn't get added to bundles unless they import from `src/typecasting.ts` directly.

It seems like the typecasting stuff is either a planned feature, or one that was removed, but I can't figure it out from the commit history. @chasers Do you know?

Reply:
Yeah that feature was WIP but can you just pull it out?  

People just need to understand that Logflare's backend is schema'd and fields have types, when you send data it needs to match that type.  Some tooling around this (client side schemas) would be better vs trying to typecast all the time.